source/ui/ui.cpp

```diff
@@ -49,23 +51,25 @@ int Ui::runUi() {
   qmlRegisterType<ClipboardAdapter>(
       "ClipboardAdapter", 1, 0, "ClipboardAdapter");
   qRegisterMetaType<std::size_t>("std::size_t");
-  qRegisterMetaType<FinalRepresentation>();
```

Comment:
Put this in again please. The ui doesn't work otherwise. I can't even properly test the pixel display.